charts/m4d-crd/templates/crds.yaml

```diff
@@ -0,0 +1,3346 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
should the folder be named `m4d-crds` rather than `m4d-crd`?

Reply:
no, in other projects such as fleet its' named `fleet-crd`

Reply:
Yeah, I saw that in fleet. Still thought that `crds` is more fitting. Doesn't really matter though.